views/index.pug

```diff
@@ -0,0 +1,84 @@
+html
+head
+    link(rel="stylesheet" href="/style.css")
+    script(src="../bagdata.js")
+    body
+     h1.title.headersCentered Faith's Bag Collection 
+     h2.headersCentered What bags are you interested in ?
+    ul.list 
+      p Choose one of the following:
+      li.listItems 
+        p 
+          strong GET 
+          | /allBags
+        div#getAll
+            p Method: GET 
+            p Route: http://localhost:2035/bags/allBags 
+            p Headers:   
+            p Description: These are all of the bags in the collection.
+      li.listItems 
+        p 
+          strong POST 
+          | /allBags
+        div 
+            p Method: POST 
+            p Route: http://localhost2035/bags/allBags
+            p Headers: Content-Type application/json
+            p Required Fields: {  name, designer, type, color,  }
+            p Description: Adds new bag data to the data provided in the body. 
+      li.listItems 
+        p 
+          strong GET 
+          | /BagsByDesigner
+        div 
+            p Method: GET 
+            p Route: http://localhost2035/bags/bagsByDesigner 
```

Comment:
You'll need the user to provide the actual designer they are looking for. Right now you don't have any way for them to do that. Something like `http://localhost2035/bags/:designer` makes more sense